adapters/rubicon/rubicon.go

```diff
@@ -552,6 +561,19 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			}
 			imp.Banner = &bannerCopy
 			imp.Video = nil
+			imp.Native = nil
+			impType = openrtb_ext.BidTypeBanner
+		} else {
+			native, err := resolveNativeObject(imp.Native, requestNative)
+			if err != nil {
+				if native == nil {
```

Comment:
here `native` will be always nil, as in `resolveNativeObject` you either `return nil, error` or `return navite, nil`